src/commands/createWebApp/WebAppCreateStep.ts

```diff
@@ -96,12 +96,27 @@ export class WebAppCreateStep extends AzureWizardExecuteStep<IWebAppWizardContex
                 default:
             }
         }
+        newSiteConfig.virtualApplications?.push({
+            virtualPath: '/',
+            physicalPath: 'site\\wwwroot',
+            preloadEnabled: true
+        })
+        // required property for setting "preloadEnabled" in VirtualApplications "true"
+        // "alwaysOn" cannot be true for "Free" tier
+        if (context?.plan?.sku?.tier !== 'Free') {
+            newSiteConfig.alwaysOn = true;
+        }
         return newSiteConfig;
     }
 
     private getAppSettings(context: IWebAppWizardContext): WebSiteManagementModels.NameValuePair[] {
         const appSettings: WebSiteManagementModels.NameValuePair[] = [];
         const disabled: string = 'disabled';
+        const trueString: string = 'true';
+        appSettings.push({
+            name: 'SCM_DO_BUILD_DURING_DEPLOYMENT',
```

Comment:
Please add a note in the changelog in the Unreleased/Changed section about this.